src/Illuminate/Validation/Validator.php

```diff
@@ -306,6 +306,22 @@ public function validate()
             throw new ValidationException($this);
         }
 
+        return $this->validated();
+    }
+
+    /**
+     * Return validated value.
```

Comment:
Change to: "Return validated values." it should be plural

Reply:
@jmarcher Thanks for review. Fixed.